HERACLES-38: Update export exclusions for Phone Call Follow-up

marta- opened the pull request:
In [HERACLES-4](https://phenotips.atlassian.net/browse/HERACLES-4): (_Implement Phone Call Follow-up form feedback_) we updated many question and section names at the client’s request, however these changes were not reflected in the field exclusions, originally implemented in [CARDS-2427](https://phenotips.atlassian.net/browse/CARDS-2427).

This PR updates the exclusions based on the new questionnaire structure.

[HERACLES-4]: https://phenotips.atlassian.net/browse/HERACLES-4?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CARDS-2427]: https://phenotips.atlassian.net/browse/CARDS-2427?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

marta- committed:
HERACLES-38: Update export exclusions for Phone Call Follow-up

sdumitriu commented on heracles-resources/feature/src/main/features/feature.json:
```diff
@@ -68,7 +68,7 @@
       "exportSchedule": "0 0 0 * * ? *",
       "fileNameFormat": "{subject}_formData_{yesterday(yyyyMMdd)}",
-      "selectors": ".bare.-labels.-identify.relativeDates.nolinks.answerFilter.answerFilter:exclude=/Questionnaires/PhoneCallFollow-Up/section_medical_care/section_provider_emergencyclinic.answerFilter:exclude=/Questionnaires/Phone Call Follow-Up/section_medical_care/section_provider_hospitalvisit.answerFilter:exclude=/Questionnaires/Phone Call Follow-Up/section_medical_care/section_provider_walk ...
+      "selectors": ".bare.-labels.-identify.relativeDates.nolinks.answerFilter.answerFilter:exclude=/Questionnaires/PhoneCallFollow-Up/section_medical_care/section_medical_care_info/section_emergency.answerFilter:exclude=/Questionnaires/Phone Call Follow-Up/section_medical_care/section_medical_care_info/section_hospital.answerFilter:exclude=/Questionnaires/Phone Call Follow-Up/section_medical_car ...
```
`PhoneCallFollow-Up` was wrong before and is still wrong, there are spaces in the node name.

sdumitriu committed:
HERACLES-38: Update export exclusions for Phone Call Follow-up
Missing spaces in a path

sashaandjic reviewed (approved):
tested

One minor thing/question out of scope of this PR: Should we be exporting hidden fields?
![image](https://github.com/user-attachments/assets/82cdda58-1413-4e0e-8356-d56edb5c113b)

marta- commented:
> One minor thing/question out of scope of this PR: Should we be exporting hidden fields?

Hidden fields should be exported by default. In prems for example, we have hidden reference questions in the survey forms with the sole purpose of including that information in the weekly exports along with the survey answers.

However, it looks like this one is a utility field and should be excluded explicitly in the configuration. Are there any others in this form?

sashaandjic commented:
> However, it looks like this one is a utility field and should be excluded explicitly in the configuration. Are there any others in this form?

No. The other two utility fields are already excluded.

marta- committed:
HERACLES-38: Update export exclusions for Phone Call Follow-up

Exclude hidden field `medical_care_type_ispredefined` which is only used internally.

marta- committed:
HERACLES-38: Update export exclusions for Phone Call Follow-up

Exclude hidden field `cancer_neuropathy` which is only used internally.

sashaandjic reviewed (approved):
tested - all good

sdumitriu reviewed (approved):
Looks good.
